config/storybook-preset-carbon/carbon-theme-addon/src/preset.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
```

Comment:
Make sure to add this to the top of CommonJS files 👀 

```suggestion
'use strict';
```